common/src/main/java/io/netty/util/internal/PlatformDependent0.java

```diff
@@ -311,6 +318,34 @@ static void copyMemory(Object src, long srcOffset, Object dst, long dstOffset, l
         }
     }
 
+    static boolean equals(byte[] bytes1, byte[] bytes2) {
```

Comment:
@trustin `Object[]` as well? Only found the code for `char[]` and it seems like they also use AVX if available [1]. Can't beat that with Unsafe :-). /cc @louiscryan 

http://hg.openjdk.java.net/jdk8u/jdk8u/hotspot/file/459a71db33dc/src/cpu/x86/vm/macroAssembler_x86.cpp#l6817